setup.py

```diff
@@ -47,6 +47,7 @@
     'pytest-pep8>=1.0.6',
     'pytest>=2.8.0',
     'pycountry>=17.5.14',
+    'PyYAML>=3.13',
```

Comment:
You probably you don't need this here since you added it already inside `install_requires`.

Reply:
@egabancho you are right, I removed the `PyYAML` and `pycountry` it was added in both places.